collection-scripts/gather_core_dumps

```diff
@@ -4,10 +4,38 @@ CORE_DUMP_PATH=${OUT:-"${BASE_COLLECTION_PATH}/node_core_dumps"}
 
 mkdir -p ${CORE_DUMP_PATH}/
 
+function get_dump_off_node { 
+    #Start Debug pod force it to stay up for 60 seconds, supress Stdout 
+    oc debug node/$1 -- /bin/bash -c 'sleep 60' > /dev/null 2>&1 &
+
+    #Get name of Debug Pod for relevant node 
+    export DEBUG=$(oc get pods --all-namespaces | grep $1-debug | awk -F' ' '{print $2}')
+
+    # Make sure pod is avaliable
+    while [ -z "${DEBUG}" ]
+    do  
+        sleep 1 
+        export DEBUG=$(oc get pods --all-namespaces | grep $1-debug | awk -F' ' '{print $2}')
```

Comment:
this looks the same as line 12. Why not just get rid of line 12 and have this loop?

Reply:
Yeah should work, there is the case the pod is already registered and we would skip this loop completely, but either should work, It just checks if "DEBUG" is empty or not